pkg/auth/rbac.go

```diff
@@ -185,6 +191,43 @@ func ValidateRbacPermission(line string) (p Permission, err error) {
 	}, nil
 }
 
+func ValidateTeamRbacPermission(line string, teamPermissions *RBACTeams) (team string, err error) {
+
+	p := strings.Split(line, ",")
+
+	if len(p) != 2 {
+		return "", fmt.Errorf("2 fields are expected but %d were specified", len(p))
+	}
+
+	users := strings.Split(p[1], " ")
+
+	if p[0] != "*" && !valid.TeamName(p[0]) {
+		return "", fmt.Errorf("invalid team name %s", p[0])
+	}
+
+	for _, user := range users {
+
+		if !valid.UserEmail(user) {
+			return "", fmt.Errorf("invalid user email %s", user)
+		}
+
+		t, ok := teamPermissions.Permissions[user]
+
+		if !ok {
+			teamPermissions.Permissions[user] = []string{p[0]}
+		} else {
+			if teamPermissions.Permissions[user][len(t)-1] == p[0] {
+				// Ignone
```

Comment:
```suggestion
				// Ignore
```
please elaborate a little bit in the comment, why we ignore this line.

Reply:
Would "Ignore if user is listed more than once in the same line/team" be more adequate?